core/src/main/java/org/openstack4j/api/networking/ext/ListenerV2Service.java

```diff
@@ -0,0 +1,69 @@
+package org.openstack4j.api.networking.ext;
+
+import org.openstack4j.common.RestService;
+import org.openstack4j.model.common.ActionResponse;
+import org.openstack4j.model.network.ext.ListenerV2;
+import org.openstack4j.model.network.ext.ListenerV2Update;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Networking (Neutron) Lbaas V2 listener Extension API
+ * @author emjburns
+ */
+public interface ListenerV2Service extends RestService {
+    /**
+     * List all listeners that the current tenant has access to
+     *
+     * @return list of all listeners
+     */
+    List<? extends ListenerV2> list();
+
+    /**
+     * Returns list of listeners filtered by parameters.
+     *
+     * @param filteringParams
+     *            map (name, value) of filtering parameters
```

Comment:
Not a big deal, saw it in many places. Existing project conventions should be fine.